recipes/pykep/meta.yaml

```diff
@@ -0,0 +1,47 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "pykep" %}
+{% set version = "2.0" %}
+{% set sha256 = "274a2053e5d85a6d5c9f60aaeeb2f70505de3b94fb5c9b06e68f92f3defaf86d" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
```

Comment:
Can you remove the comments from the example recipe?

Reply:
sure